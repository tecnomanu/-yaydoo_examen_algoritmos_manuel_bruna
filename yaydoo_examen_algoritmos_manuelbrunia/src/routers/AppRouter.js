import React from 'react'
import { BrowserRouter, Route, Routes } from 'react-router-dom'
import { HomeScreen } from '../components/home/HomeScreen'
import { Test1Screen } from '../components/test-1/Test1Screen'
import { Test2Screen } from '../components/test-2/Test2Screen'
import { Navbar } from '../components/ui/NavBar'

export const AppRouter = () => {
    return (
        <BrowserRouter>
            <Navbar />
            <Routes>
                <Route path="/" element={<HomeScreen></HomeScreen>}></Route>
                <Route path="/test-1" element={<Test1Screen></Test1Screen>}></Route>
                <Route path="/test-2" element={<Test2Screen></Test2Screen>}></Route>
            </Routes>
        </BrowserRouter>
    )
}