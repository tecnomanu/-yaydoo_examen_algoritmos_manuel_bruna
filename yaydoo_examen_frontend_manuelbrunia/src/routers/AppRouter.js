import React from 'react'
import { BrowserRouter, Route, Routes } from 'react-router-dom'
import { HomeScreen } from '../components/home/HomeScreen'
import { UsersScreen } from '../components/users/UsersScreen'
import { Navbar } from '../components/ui/NavBar'

export const AppRouter = () => {
    return (
        <BrowserRouter>
            <Navbar />
            <Routes>
                <Route path="/" element={<HomeScreen></HomeScreen>}></Route>
                <Route path="/users" element={<UsersScreen></UsersScreen>}></Route>
                {/* <Route path="/users/create" element={<Test2Screen></Test2Screen>}></Route> */}
            </Routes>
        </BrowserRouter>
    )
}