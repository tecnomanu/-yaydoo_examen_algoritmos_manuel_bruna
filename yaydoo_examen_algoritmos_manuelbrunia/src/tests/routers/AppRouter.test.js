import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import { AppRouter } from '../../routers/AppRouter';

describe("Test on <AppRouter/>", ()=>{
    test('SnapShot of <AppRouter/>', ()=>{
        const wrappper = shallow(<AppRouter />);

        expect(wrappper).toMatchSnapshot();
    })
})