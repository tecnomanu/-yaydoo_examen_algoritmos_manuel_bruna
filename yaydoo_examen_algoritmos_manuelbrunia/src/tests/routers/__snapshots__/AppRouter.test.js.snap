// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <AppRouter/> SnapShot of <AppRouter/> 1`] = `
<BrowserRouter>
  <Navbar />
  <Routes>
    <Route
      element={<HomeScreen />}
      path="/"
    />
    <Route
      element={<Test1Screen />}
      path="/test-1"
    />
    <Route
      element={<Test2Screen />}
      path="/test-2"
    />
  </Routes>
</BrowserRouter>
`;
